server/routes/staticContent/indexPresenter.ts

```diff
@@ -0,0 +1,11 @@
+export default class StaticContentIndexPresenter {
+  constructor(private readonly pages: { path: string; template: string; description: string }[]) {}
+
+  get rows(): { href: string; template: string; description: string }[] {
```

Comment:
Something I'm not sure about is, what are people's opinions on creating interfaces for arguments & return types for this sort of thing? I always tend to create an interface rather than return plain objects, but I don't know if that's over the top.

Reply:
It's a good question - I usually don't bother creating a type when it's something that's only going to be passed between a couple of places, but it's possibly just laziness of not wanting to think about where to put the type or what to name it. I'll keep it in mind for the future though and possibly use more types 😁